src/dnn/helpers.jl

```diff
@@ -115,3 +115,16 @@ function PoolDesc(nd, window, padding, stride, mode, maxpoolingNanOpt=CUDNN_NOT_
     finalizer(free, this)
     return this
 end
+
+mutable struct ActivationDesc; ptr; end
```

Comment:
Why is this mutable & untyped? It doesn't look like `ptr` is ever assigned to, let alone changing types.

Reply:
I was simply following the convention as in `ConvDesc` and `PoolDesc`.

Reply:
Yeah, I noticed. Should be improved, but can happen in a different PR. Do fix the `Ref` stuff though, because I recently went through and fixed existing code.